app/controllers/application.js

```diff
@@ -49,6 +49,18 @@ export default Controller.extend(cancelOrderMixin, {
   cartscroll: service(),
 
   hasCartItems: alias("cart.isNotEmpty"),
+
+  hasValidCartItems: computed(
```

Comment:
was thinking can we have this computed property in `cart` service itself. This way we can just use
 `cartService. hasValidCartItems`. Thoughts ?